lntest/itest/lnd_test.go

```diff
@@ -3044,8 +3045,8 @@ func testChannelForceClosure(net *lntest.NetworkHarness, t *harnessTest) {
 	// output spending from the commitment txn, so we must deduct the number
 	// of blocks we have generated since adding it to the nursery, and take
 	// an additional block off so that we end up one block shy of the expiry
-	// height.
-	cltvHeightDelta := defaultCLTV - defaultCSV - 2 - 1
+	// height, and add one additional block for the block padding.
+	cltvHeightDelta := uint32(routing.BlockPadding) + defaultCLTV - defaultCSV - 2 - 1
```

Comment:
perhaps define a small helper for padding cltvs, e.g. `padCLTV(x) { return x+routing.BlockPadding }`?